README.md

```diff
@@ -45,6 +45,8 @@ Parameters (specified as one object passed into hot-shots):
 * `useDefaultRoute`: Use the default interface on a Linux system. Useful when running in containers
 * `protocol`: Use `tcp` option for TCP protocol, or `uds` for the Unix Domain Socket protocol. Defaults to UDP otherwise
 * `path`: Used only when the protocol is `uds`. Defaults to `/var/run/datadog/dsd.socket`.
+* `udsGracefulErrorHandling`: Used only when the protocol is `uds`. Boolean indicating wether to handle socket errors gracefully. defaults falsey
```

Comment:
```suggestion
* `udsGracefulErrorHandling`: Used only when the protocol is `uds`. Boolean indicating whether to handle socket errors gracefully.  Defaults to true.
```

Note the suggestion above to default to true- it sounds like this is really broken without this being enabled, correct?  Any problems with having it default to true